nlp_primitives/tests/test_universal_sentence_encoder.py

```diff
@@ -1,13 +1,15 @@
 import numpy as np
 import pandas as pd
+import pytest
 
 from ..universal_sentence_encoder import UniversalSentenceEncoder
-from ..utils import PrimitiveT  # find_applicable_primitives, valid_dfs
+from ..utils import PrimitiveT
 
 
 class TestUniversalSentenceEncoder(PrimitiveT):
     primitive = UniversalSentenceEncoder
 
+    @pytest.mark.skip()
```

Comment:
When docstring tests are setup, `test_regular` starts to raise this error on Circle CI.

```log
...
nlp_primitives/tests/test_title_word_count.py::TestTitleWordCount::test_arg_init <- nlp_primitives/utils.py PASSED [ 85%]
nlp_primitives/tests/test_universal_sentence_encoder.py::TestUniversalSentenceEncoder::test_regular make: *** [Makefile:16: unit-tests] Killed
Exited with code 2
CircleCI received exit code 2
```

If this test is skipped, then the error isn't raised anymore.

```log
...
nlp_primitives/tests/test_title_word_count.py::TestTitleWordCount::test_arg_init <- nlp_primitives/utils.py PASSED [ 85%]
nlp_primitives/tests/test_universal_sentence_encoder.py::TestUniversalSentenceEncoder::test_regular SKIPPED [ 87%]
nlp_primitives/tests/test_universal_sentence_encoder.py::TestUniversalSentenceEncoder::test_name_and_desc <- nlp_primitives/utils.py PASSED [ 88%]
...
==================== 70 passed, 1 skipped in 87.09 seconds =====================
CircleCI received exit code 0
```

Not sure why this is case.

Reply:
This was resolved in 4c5dfb9ba9e8e50b1fb30ee6ee1afa3b96d0b225 by sharing the primitive as a fixture across the test session.